docker-compose.yml

```
@@ -72,10 +72,10 @@ x-environment: &default-environment
   # Drupal Shield credentials.
   DRUPAL_SHIELD_USER: ${DRUPAL_SHIELD_USER:-}
   DRUPAL_SHIELD_PASS: ${DRUPAL_SHIELD_PASS:-}
-  #;< SERVICE_REDIS
-  # Redis integration flag.
-  DRUPAL_REDIS_ENABLED: ${DRUPAL_REDIS_ENABLED:-}
-  #;> SERVICE_REDIS
+  #;< SERVICE_VALKEY
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)